FMI-CROSS-CHECK-RULES.md

```diff
@@ -225,9 +199,8 @@ Vendors of tools that import FMUs that provide access to licensed tools and woul
 
 To submit per imported FMU:
 
-- `{Model_Name}_out.csv`: Computed results as a CSV file for the same variables as given in the reference CSV file.
-
-- In order to classify the result as `passed`, the results should correspond to the reference solution.
+- ~~`{Model_Name}_out.csv`: Computed results as a CSV file for the same variables as given in the reference CSV file.~~
+This requirement was dropped to reduce the space requirement of the repository.
```

Comment:
Would it be ok to make this a "may" rule, like "importers may support a link to csv result files that prove the self assessment"?

Reply:
I think, that importing tool vendors will take this more seriously, if the results can be checked by everyone.
Thus it increases the the credibility of the cross-check.
Thus I am in favour of a "must" rule. 
What are arguments against (except for a little bit more work)? The tool vendors must simulate the FMUs, create the results and compare the  them with the reference anayway.

Reply:
This is quite a bit more work because it requires picking another place where to put the files, keeping them current end so on. The idea is that whoever wants to cheat can. Not much we can do about this, unless someone asks us to investigate. Then we need to get the results from the importing tool vendor and run an investigation. How much would it help to have the results available up front?